addons/calendar/models/calendar_event.py

```diff
@@ -528,7 +528,7 @@ def action_sendmail(self):
         email = self.env.user.email
         if email:
             for meeting in self:
-                meeting.attendee_ids._send_mail_to_attendees('calendar.calendar_template_meeting_invitation')
+                meeting.attendee_ids._send_mail_to_attendees(self.env.ref('calendar.calendar_template_meeting_invitation'))
```

Comment:
You should probably add `raise_if_not_found=False` to all those calls to only get a warning because the user is not notified rather than a full raised error.
That will be closer to previous behavior.

Reply:
done